SDF-to-Coords

```diff
@@ -0,0 +1,31 @@
+import os
+
+def SDF_to_Coords(filename, num_atom):
+    directory = r'/mnt/d/SDFs/structures/'
+    for filename in os.listdir(directory):
+        if filename.endswith(".sdf"):
+            with open(filename) as f:
+                xyz = []
+                counter = 0
+                for line in f:
+                    counter += 1
+                    if counter < 4:
+                        continue
+                      
+                    elif counter > 4 and counter <= 4+num_atom:
+                        xyz.append(line.split("   ")[1:4])
```

Comment:
In making the xyz list, this code cannot separate the elemental symbol from the z coordinate for each atom. I was thinking that it needs more parsing, but the things I have tried have not panned out. In such cases I don't know if the idea is wrong, or the execution is wrong. Execution probably, idea maybe.

Reply:
One thing I would recommend is checking in to the repo at least one of the SDF files to make it easier for me to see what you are parsing. That said try just using `line.split()` (no argument) I believe that will do what you want (I think it parses 
```
1 2  3
``` 
into `['1', '2', '3']` and not `['1', '2', '', '3']`. Even If it doesn't, it's in my opinion easier to loop over the `['1', '2', '', '3']` array and throw away the elements you don't want than it is to do two rounds of parsing.